diff --git a/symengine/tests/basic/test_number.cpp b/symengine/tests/basic/test_number.cpp
index 12b58cf3..8920e256 100644
--- a/symengine/tests/basic/test_number.cpp
+++ b/symengine/tests/basic/test_number.cpp
@@ -6,6 +6,7 @@
 #include <symengine/eval_mpc.h>
 #include <symengine/eval_mpfr.h>
 #include <symengine/symengine_exception.h>
+#include <symengine/pow.h>
 
 using SymEngine::SymEngineException;
 using SymEngine::print_stack_on_segfault;
diff --git a/symengine/tests/basic/test_sets.cpp b/symengine/tests/basic/test_sets.cpp
index 134fac1b..c3e1b697 100644
--- a/symengine/tests/basic/test_sets.cpp
+++ b/symengine/tests/basic/test_sets.cpp
@@ -4,6 +4,7 @@
 #include <symengine/logic.h>
 #include <symengine/infinity.h>
 #include <symengine/symengine_exception.h>
+#include <symengine/pow.h>
 
 using SymEngine::Basic;
 using SymEngine::Integer;
