diff --git a/symengine/prime_sieve.cpp b/symengine/prime_sieve.cpp
index 97ed4837..d04910c5 100644
--- a/symengine/prime_sieve.cpp
+++ b/symengine/prime_sieve.cpp
@@ -1,6 +1,8 @@
 #include <symengine/prime_sieve.h>
+#include <ciso646>
 #include <cmath>
 #include <valarray>
+#include <algorithm>
 
 namespace SymEngine
 {
diff --git a/symengine/printers/stringbox.cpp b/symengine/printers/stringbox.cpp
index afa1f115..af23b3e8 100644
--- a/symengine/printers/stringbox.cpp
+++ b/symengine/printers/stringbox.cpp
@@ -1,5 +1,6 @@
 #include <iterator>
 #include <sstream>
+#include <algorithm>
 #include <symengine/printers/stringbox.h>
 
 namespace SymEngine
